passage/src/main/java/id/passage/android/Passage.kt

```diff
@@ -653,5 +655,76 @@ public final class Passage(
         if (!isUsingTokenStore) return
         tokenStore.setTokens(authResult)
     }
+
+    /**
+     * Updates the Passage Token Store with the given ID token.
+     *
+     * This method should be called whenever a user utilizes Hosted Auth to ensure the
+     * isToken is updated in the Passage Token Store, if applicable.
+     *
+     * @param idToken The ID token to be handled.
+     */
+
+    private fun handleIdToken(idToken: String) {
+        if (!isUsingTokenStore) return
+        tokenStore.setIdToken(idToken)
+    }
+    // endregion
+
+    // region OIDC Methods
+
+    /**
+     * Authentication Method for Hosted Apps
+     *
+     * If your Passage app is Hosted, use this method to register and log in your user.
+     * This method will open up a Passage login experience on a Chrome tab.
+     */
+    public fun hostedAuthStart() {
+        PassageHosted.openChromeTab(
+            activity,
+        )
+    }
+
+    /**
+     * Finish Hosted Auth for Hosted Apps
+     *
+     * This method completes the hosted authentication process by exchanging the provided authorization code for Passage tokens.
+     *
+     * @param code The code returned from app link redirect to your activity.
+     * @param clientSecret You hosted app's client secret, found in Passage Console's OIDC Settings.
+     * @param state The state returned from app link redirect to your activity.
+     * @throws HostedAuthorizationError
+     */
+
+    suspend fun hostedAuthFinish(
+        code: String,
+        clientSecret: String,
+        state: String,
+    ): Pair<AuthResult, String> {
```

Comment:
To be consistent with our other public facing auth methods, I suggest:
* Adding an explicit `public` declaration
* Changing `AuthResult` to `PassageAuthResult` (its a just type alias, left over from some legacy changes)
```suggestion
    public suspend fun hostedAuthFinish(
        code: String,
        clientSecret: String,
        state: String,
    ): Pair<PassageAuthResult, String> {
```